bot/cogs/grades_data/grades_cog.py

```diff
@@ -164,7 +167,12 @@ def initialize(self, year):
             if os.path.isfile(f'bot/cogs/grades_data/assets/master_prof-{year}.json'):
                 with open(f'bot/cogs/grades_data/assets/master_prof-{year}.json', 'r') as f:
                     prof = json.load(f)
-            return [normal, prof]
+
+            if os.path.isfile(f'bot/cogs/grades_data/assets/master_prof.json'):
+                with open(f'bot/cogs/grades_data/assets/master_prof.json', 'r') as f:
+                    totalProf = json.load(f)
+                    
+            return [normal, prof, totalProf]
```

Comment:
semantically this should return a tuple, list implies mutability and this is used as an immutable object